### ISSUE TITLE
user cluster failed to pull velero image

### ISSUE THREAD

**Issue by stoneshi-yunify** [jibutech/backup-saas-client#13]

**Comment by shaofeng66** [comment]
We now switch to aliyun registry to accelerate the image pulling. For those clusters that have no access to internet, they will have to manually setup private registry and jibu need some code change to support this scenario. Please let us know if this is a requirement?

**Comment by stoneshi-yunify** [comment]
yes we definitely need to support this scenario but this is can be deferred to phase 2
